src/js/hca-rjsf/components/IntroductionPage.jsx

```diff
@@ -27,32 +59,27 @@ class IntroductionPage extends React.Component {
         </p>
         <div className="usa-alert usa-alert-info">
           <div className="usa-alert-body">
-            <strong>You won’t be able to save your work or come back to finish.</strong> So before you start, it’s a good idea to gather information about your military and education history, and the school you want to attend.
+            {this.getAlertBody()}
           </div>
         </div>
         <br/>
-        <div className="row progress-box progress-box-schemaform form-progress-buttons schemaform-buttons">
-          <div className="small-6 usa-width-five-twelfths medium-5 columns">
-            <a href="/healthcare/apply">
-              <button className="usa-button-outline">« Back</button>
-            </a>
-          </div>
-          <div className="small-6 usa-width-five-twelfths medium-5 end columns">
-            <ProgressButton
-                onButtonClick={this.goForward}
-                buttonText="Continue"
-                buttonClass="usa-button-primary"
-                afterText="»"/>
-          </div>
-        </div>
-        <div className="omb-info--container">
+        <FormIntroButtons route={this.props.route}/>
+        <br/>
+        {/* TODO: Remove inline style after I figure out why .omb-info--container has a left padding */}
+        <div className="omb-info--container" style={{ paddingLeft: 0 }}>
           <OMBInfo resBurden={30} ombNumber="2900-0091" expDate="05/31/2018"/>
         </div>
       </div>
     );
   }
 }
 
-export default withRouter(IntroductionPage);
+function mapStateToProps(state) {
+  return {
+    loggedIn: state.user.login.currentlyLoggedIn
+  };
+}
+
+export default connect(mapStateToProps)(IntroductionPage);
```

Comment:
Yeah, it would also be nice to figure out how to share more of this code between the different forms. This is sort of tedious to do for each form.

Maybe some kind of higher order component that passes the info box thing to an intro page component, so it can put it wherever it wants?